MAINTAINERS.md

```diff
@@ -0,0 +1,16 @@
+## Project Maintainers
+
+Below is the list of maintainers for this repo (in alphabetical order). For SMEs
+on particular package, please see the [CODEOWNERS file](CODEOWNERS).
+
+- Biniam Admikew ([@b-admike](https://github.com/b-admike))
+- Diana Lau ([@dhmlau](https://github.com/dhmlau))
+- Dominique Emond ([@emonddr](https://github.com/emonddr))
+- Hugo Da Roit ([@Yaty](https://github.com/Yaty))
+- Janny Hou ([@jannyhou](https://github.com/jannyhou))
+- Mario Estrada ([@marioestradarosa](https://github.com/marioestradarosa))
+- Miroslav Bajtos ([@bajtos](https://github.com/bajtos))
+- Nora Abdelgadir ([@nabdelgadir](https://github.com/nabdelgadir))
+- Raymond Feng ([@raymondfeng](https://github.com/raymondfeng))
+- TN ([@thinusn](https://github.com/thinusn))
+- Yaapa Hage ([@hacksparrow](https://github.com/hacksparrow))
```

Comment:
Maybe we should divide the list into IBMers and community members. Or mark us as IBMers.

Reply:
How about `core maintainers` and `community maintainers` that are used somewhere else in the docs? e.g. https://loopback.io/doc/en/contrib/Governance.html.